Edit first-integration, navigating, dashboard (#1)
spinningrachel/cycode: docs/dashboard.md

# The Dashboard  

> After logging in, you are taken to the dashboard. To get to the dashboard when you are in another area of Cycode, click on the Cycode or dashboard icons. Along the top of the dashboard there are:  

Reviewer: ```suggestion
Once you've logged in, the dashboard loads. 

To navigate to the dashboard when you are in another area of Cycode, click the Cycode icon from the top, or click <enter icon here> from the left side panel. 

The dashboard appears similar to the following image:
<enter image of dashboard. obfuscate personal and company information>

Along the top of the dashboard, find these cards:  
```

> - users: anyone with access to the pipeline   

Reviewer: @rgottesman  Since all of these are parts of the screen, they are proper names and should be capitalized.

- contributors: anyone that contributes to the platform  

- teams: groups who contribute to the pipeline. Different teams have different roles and permissions  

- repositories: a list of the repositiries you are tracking  

> - organizations: ?

Reviewer: these are the organizations managed from the third-party app - hierarchically, they are above the repos, teams and contributors.

> Click on them to see more detailes and manage them.  

Reviewer: ```suggestion
Click any of these cards to see more details and manage them.  Cycode redirects you to the relevant area. 
```

> Below that, there are two lists:  

Reviewer: ```suggestion
Below these cards, there are two lists:  
```

@rgottesman isn't specific since it's far away from the initial reference of what you're talking about.

> -  act: notification of weaknesses in your credentials that you need to deal with in the near future  

Reviewer: you need to deal with in the near future  - not urgent enough; you can use the word "act" - for example you need to act on quickly

> - improve: notification of general improvements you need to make for security  

Reviewer: ```suggestion
- improve: notification of best practices you should implement in order to improve your security posture
```

> Below that, there are three graphs or charts showing alerts:  

Reviewer: ```suggestion
Below that, there are three graphs or charts showing alerts:  
```

@rgottesman instead of Below that over and over again, add number stamps to the original image of the dashboard (that I suggested you add) and then insert those numbers as images next to the description of each part (like I demo'd for you today when I was showing you how to insert images)

- alerts over time: a graph showing the number of alerts over the past seven months, in groups of thirty days. The alerts are divided by severity in each group of days.  

- alerts by severity: a chart that groups current alerts by severity.

> - alerts by type:  a chart that groups current alerts by type. Clicking on a type will take you to a full list of those alerts.

Reviewer: ```suggestion
- alerts by type:  a chart that groups current alerts by type. Click any of the types to navigate to the full list of the relevant alerts.
```

@rgottesman try to avoid gerunds

> At the bottom of the page, there are three sections:  

Reviewer: @rgottesman again - number stamps I think are preferable for the layout

also - for all of the lists, i just realized you're using colons where i would have used dashes - if you could default to dashes that would be great b/c i won't ever remember to change my habits at this stage in the game 

it probably feels awkward to you b/c you're using dashes in the markdown for bullets - but remember the bullets convert to nice GUI bullets in teh output :)

Author: @spinningrachel Which dash should I use? Hyphen, en dash, or em dash?

Reviewer: @rgottesman in markdown for our use cases, you can always use just the regular hyphen and expect it to format correctly in the output
